tests/unit/GithubUtilsTest.js

```diff
@@ -533,7 +534,7 @@ describe('GithubUtils', () => {
         ));
 
         test('Test some verified internalQA PRs', () => (
-            githubUtils.generateStagingDeployCashBody(tag, [...basePRList, ...internalQAPRList], [internalQAPRList[0]])
+            githubUtils.generateStagingDeployCashBody(tag, [...basePRList, ...internalQAPRList], [], [], [], [], [internalQAPRList[0]])
```

Comment:
We need to pass this new argument containing the resolved/verified InternalQA PR to the function in the test to make sure that the checkbox state has been preserved.

Reply:
The reason this test has been passing before (although this feature clearly does not work) is that we have not been simulating the actual flow as it happens in reality. We have passed in the checked-off internalQA PR as a third argument, which then made it to the [verifiedOrNoQAPRs](https://github.com/Expensify/App/blob/27031e108f9f76486c456198b7448fbbbc3afa43/.github/libs/GithubUtils.js#L251) array which then worked, however, normally, the internalQA PR would never make it to the third argument 😢

Reply:
Okay, makes sense 👍